tests/unit/test_published_clusters_with_data.py

```diff
@@ -0,0 +1,51 @@
+import responses
+
+from tamr_unify_client import Client
+from tamr_unify_client.auth import UsernamePasswordAuth
+
+
+@responses.activate
+def test_published_clusters_with_data():
+    project_config = {
+        "name": "Project 1",
+        "description": "Mastering Project",
+        "type": "DEDUP",
+        "unifiedDatasetName": "Project 1 - Unified Dataset",
+        "externalId": "Project1",
+        "resourceId": "1",
+    }
+
+    unified_dataset_json = {
+        "id": "unify://unified-data/v1/datasets/8",
+        "name": "Project_1_unified_dataset",
+        "version": "10",
+        "relativeId": "datasets/8",
+        "externalId": "Project_1_unified_dataset",
+    }
+
+    rcwd_json = {
```

Comment:
`rcwd` is RecordClustersWithData, which is not what you're working with; this should PublishedClustersWithData, or `pcwd`.